client/src/components/Chat/ChatDisplayArea/ChatDisplayArea.jsx

```diff
@@ -1,14 +1,23 @@
-import chats from "../InputArea/chatsDummyDataGenerator.js";
+import React from "react";
+import { v4 as uuidv4 } from "uuid";
 import MessageBox from "../MessageBox/MessageBox.jsx";
+import { useSelector } from "react-redux";
 import { ChatsContainer } from "../../../styles/Chat/ChatDisplay/ChatsContainer.jsx";
 
-export default function ChatDisplayArea() {
+const ChatDisplayArea = ({ messages }) => {
+  //! must be refactored, when a loggedUser slice is created
+  const { name: loggedUser } = useSelector(
+    (state) => state.userRegister.connectedUser
+  );
+  console.log("loggedUser", loggedUser, "loggedUser");
+
   return (
     <ChatsContainer>
-      {chats.map((message, index) => (
-        // could be sorted by timestamp before display -not added here-
-        <MessageBox message={message} key={index} />
+      {messages.map((message) => (
+        <MessageBox message={message} loggedUser={loggedUser} key={uuidv4()} />
```

Comment:
bad practice using rand generated id

Reply:
but i used the uuid based on the article you sent in slack 
https://react.dev/learn/rendering-lists#where-to-get-your-key

"Locally generated data: If your data is generated and persisted locally (e.g. notes in a note-taking app), use an incrementing counter, [crypto.randomUUID()](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) or a package like [uuid](https://www.npmjs.com/package/uuid) when creating items."